src/api/payment-requests/payment-requests.md

```diff
@@ -539,6 +539,12 @@ Alternatively you can provide an external transaction Id and the Centrapay [Asse
 }
 {% endjson %}
 
+{% h4 Error Responses %}
+
+| Status | Code                    | Description                                                                                                                                              |
+|:-------|:------------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 403    | INVALID_ASSET_TYPE      | Either the merchant is not configured with the provided asset type, or the asset type can not be used as it is only supported by our [Legacy Payment API][]. |
```

Comment:
There's another case where the asset type just doesn't exist.

Reply:
Good point, I've just added that in now.